eo-maven-plugin/src/main/java/org/eolang/maven/PullMojo.java

```diff
@@ -64,24 +66,17 @@ public final class PullMojo extends SafeMojo {
      * The Git hash to pull objects from, in objectionary.
      *
      * @since 0.21.0
+     * @checkstyle VisibilityModifierCheck (6 lines)
      */
     @SuppressWarnings("PMD.ImmutableField")
     @Parameter(property = "eo.tag", required = true, defaultValue = "master")
     private String tag = "master";
 
-    /**
-     * Pull again even if the .eo file is already present?
-     *
-     * @checkstyle MemberNameCheck (7 lines)
-     * @since 0.10.0
-     */
-    @Parameter(property = "eo.overWrite", required = true, defaultValue = "false")
-    private boolean overWrite;
-
     /**
      * Read hashes from local file.
      *
      * @checkstyle MemberNameCheck (7 lines)
+     * @checkstyle VisibilityModifierCheck (6 lines)
```

Comment:
@maxonfjvipon Same here. Could you please explain why we need to exclude the `VisibilityModifierCheck` here?